templates/azure-functions/public/staticwebapp.config.json

```
@@ -0,0 +1,18 @@
+{
+  "routes": [
+    {
+      "route": "/",
+      "rewrite": "/api/azure"
+    },
+    {
+      "route": "/index.html",
+      "rewrite": "/api/azure"
+    }
+  ],
```

With this setting POST requests to sub-path routes do not work (see https://github.com/remix-run/remix/pull/2521#issuecomment-1156008014). 

It would be better to have a general fallback to the Azure Function like this:

```suggestion
  "routes": [
    {
      "route": "/build/*"
    },
    {
      "route": "/favicon.ico"
    },
    {
      "route": "/*",
      "rewrite": "/api/azure"
    }
  ],
```